gulpfile.js

```diff
@@ -108,6 +108,7 @@ gulp.task('copy-static-files', [
   'copy-test-dirs',
   'copy-images',
   'copy-bootstrap',
+  'copy-fontawesome',
   'copy-webmaker-app-icons'
```

Comment:
Shouldn't `copy-webmaker-app-icons` and its task definition further down in this file be removed?